public/app/user-service-registry/user-service-registry.controller.js

```diff
@@ -68,6 +68,7 @@
             resetServiceInputArrays();
             var i;
             for (i=0;i<inputs.length;i++){
+                inputs[i].formatSelect = inputs[i].formats[0].mimeType;
```

Comment:
Are there going to be more formats or is the first in the list all that matters?